specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -3409,6 +3409,15 @@
             "read"
           ]
         },
+        "serverlessComputeCustomSubnet": {
+          "description": "The ARM resource ID for the subnet under which to create serverless compute clusters",
+          "type": "string",
+          "format": "arm-id"
+        },
+        "serverlessComputeNoPublicIP": {
+          "description": "The flag to signal if serverless compute nodes deployed in custom vNet would have no public IP address when inside workspace with no public network access",
```

Comment:
"The flag to signal if serverless compute nodes deployed in custom vNet would have no public IP addresses for a workspace with private endpoint"